ui/app/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -65,6 +65,41 @@ export default function WhatsNewPopup({ onClose }) {
     [memoizedNotifications],
   );
 
+  const maybeRenderNotificationItem = (id, isFirstNotification, property) => {
+    const notification = UI_NOTIFICATIONS[id];
+    if (isFirstNotification && property === 'image') {
+      return (
+        <img
+          className="whats-new-popup__notification-image"
+          src={notification[property]}
+        />
+      );
+    }
+
+    if (isFirstNotification && property === 'actionText') {
+      return (
+        <Button
+          type="secondary"
+          className="whats-new-popup__button"
+          rounded
+          onClick={actionFunctions[id]}
+        >
+          {t(notification[property])}
```

Comment:
General reminder that we should avoid using variables as the first parameter to the `t` function. Instead we should always use string literals, which are easier for our localized message linter to detect.

Reply:
Addressed in 763ad6d66